doc/README.md

```diff
@@ -79,6 +79,7 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 - [Reduce Memory](reduce-memory.md)
 - [Reduce Traffic](reduce-traffic.md)
 - [Tor Support](tor.md)
+- [I2P Support](i2p.md)
```

Comment:
nit, sort (these are listed in alphabetical order, not by topic)

Reply:
agree on sorting `I2P Support`

Sorting `Init Scripts` and the `PSBT` entries can be left for a followup

Reply:
Sorted.